Test/test.js

```diff
@@ -7,6 +7,80 @@ const app = require('../server/server.js');
 const should = chai.should();
 chai.use(chaiHttp);
 
+describe('/POST description', () => {
```

Comment:
This should probably also address what happens if someone tries to `POST` something in the wrong shape to make sure that an error on the `POST` doesn't gum up the works.